cvmfs/cvmfs.cc

```diff
@@ -1829,123 +1830,128 @@ static int Init(const loader::LoaderExports *loader_exports) {
   cvmfs::backoff_throttle_ = new BackoffThrottle();
 
   // Option parsing
-  options::Init();
+  if (cvmfs::loader_exports_->simple_options_parsing) {

```

Comment:
The naming should be consistent, it's probably best to use "Simple" or "simple" everywhere instead of "Fast" or "fast".